_data/investing.yml

```
@@ -172,9 +172,12 @@ websites:
 
     - name: TD Waterhouse
```

Looks like they're not using "Waterhouse" in their name any longer.
```suggestion
    - name: TD Direct Investing
```